### PR TITLE
Euwe 2 announce

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
@mfeifer can you review. 😺

**Comment by JPrause** [comment]
@simaishi can you also give this a quick review.

File: site/_posts/2017-03-06-Announcing-Euwe-2.html.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+and here are just a few of the things added since Euwe-1 release:
+- Automate: Automate Retry with Server Affinity
+- Providers:
+  * Ansible Tower: Advanced search for Ansible Tower Jobs not visible on switch from a different tab
+  * Microsoft Azure: Delete all resources when deleting an Azure stack
+
+- Fixed issues with User Interface (Classic)
+  * Fix missing Smart State Analysis button on Cloud Instances list view
+  * Red Hat Enterprise Virtualization: Removed the option to migrate the VMs outside of the cluster. 
+
```
Comment (mfeifer) [inline site/_posts/2017-03-06-Announcing-Euwe-2.html.md]: I'd replace lines 23 - 25 with

Notable Platform fixes include:
 - Tenant admin should not be able to create groups in other tenants
 -  Introduce report result purging timer

File: site/_posts/2017-03-06-Announcing-Euwe-2.html.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Here is the changelog:
```
Comment (mfeifer) [inline site/_posts/2017-03-06-Announcing-Euwe-2.html.md]: Here is the complete changelog:

**Comment by JPrause** [comment]
@cybette we go the all-clear from QE about smoke testing,..so we can merge this announcement.

**Pull request merged**